apps/service_providers/messaging_service.py

```diff
@@ -183,17 +185,57 @@ def send_voice_message(
     ):
         # OGG must use the opus codec: https://whatsapp.turn.io/docs/api/media#uploading-media
         voice_audio_bytes = synthetic_voice.get_audio_bytes(format="ogg", codec="libopus")
-        media_id = self.client.media.upload_media(voice_audio_bytes, content_type="audio/ogg")
-        self.client.messages.send_audio(whatsapp_id=to, media_id=media_id)
+        audio_file = BytesIO(voice_audio_bytes)
+        audio_file.name = "voice_message.ogg"
+
+        self.client.messages.send_media(
+            whatsapp_id=to, file=audio_file, content_type="audio/ogg", media_type="audio", caption=None
+        )
 
     def get_message_audio(self, message: TurnWhatsappMessage) -> BytesIO:
         response = self.client.media.get_media(message.media_id)
         ogg_audio = BytesIO(response.content)
         return audio.convert_audio(ogg_audio, target_format="wav", source_format="ogg")
 
     def can_send_file(self, file: File) -> bool:
-        # When support for Turn.IO is added, this should be updated
-        return False
+        mime = file.content_type
+        size = file.content_size or 0  # in bytes
+
+        if mime is None:
+            return False
+
+        if mime.startswith("image/"):
+            return size <= 5 * 1024 * 1024  # 5 MB
+        elif mime.startswith(("video/", "audio/")):
+            return size <= 16 * 1024 * 1024  # 16 MB
+        elif mime.startswith("application/"):
+            return size <= 100 * 1024 * 1024  # 100 MB
+        else:
+            return False
+
+    def send_file_to_user(self, from_: str, to: str, platform: ChannelPlatform, file: File, download_link: str):
+        file_name = file.name
+        mime_type, _ = mimetypes.guess_type(file_name)
```

Comment:
Since this is a `File` instance, the `content_type` attribute should already be set which does the same as you're doing here, so you can just use that